nbviewer/templates/faq.md

```diff
@@ -4,7 +4,7 @@
 
 <div class="col-md-10 col-md-offset-1">
 
-{% filter markdown(extensions=['headerid(level=3)','toc'], extension_configs= {'toc' : [('anchorlink', True)]}) %}
+{% filter markdown(extensions=['toc'], extension_configs= {'toc' : [('anchorlink', True)]}) %}
```

Comment:
We should probably specify `markdown>=X` in the `requirements.txt` where X is some minimum version known to work after this code change. The current version you have installed for local dev and test should work. Could you please add it to `requirements.txt` in this PR?

Reply:
That's a good idea. I added it to `requirements.txt`.